js/survery-app.js

```diff
@@ -0,0 +1,2 @@
+import html from '../html.js';
+import surveyApi from
```

Comment:
```suggestion
import surveyApi from 'survey-api.js';
```
Currently your api file is named 'survey.api.js', you'll want to change that to something like 'survey-api.js'.